components/SizerBase/src/SizerBase.xaml

```diff
@@ -49,33 +66,58 @@
-                                        <Setter Target="RootGrid.Background" Value="{ThemeResource GripperBarBackgroundPressedOver}" />
+                                        <Setter Target="RootGrid.Background" Value="{ThemeResource SizerBaseBackgroundPressed}" />
+                                    </VisualState.Setters>
+                                </VisualState>
+                                <VisualState x:Name="Disabled">
+                                    <VisualState.Setters>
+                                        <Setter Target="RootGrid.Background" Value="{ThemeResource SizerBaseBackgroundDisabled}" />
+                                        <Setter Target="PART_Indicator.Opacity" Value="0.45" />
+                                    </VisualState.Setters>
+                                </VisualState>
+                            </VisualStateGroup>
+                            <VisualStateGroup x:Name="OrientationStates">
+                                <VisualState x:Name="Vertical" />
+                                <VisualState x:Name="Horizontal">
+                                    <VisualState.Setters>
+                                        <Setter Target="PART_Indicator.Width" Value="{ThemeResource SizerBaseIndicatorHeight}" />
+                                        <Setter Target="PART_Indicator.Height" Value="{ThemeResource SizerBaseIndicatorWidth}" />
+                                    </VisualState.Setters>
+                                </VisualState>
+                            </VisualStateGroup>
+                            <VisualStateGroup x:Name="IndicatorVisibilityStates">
+                                <VisualState x:Name="Visible" />
+                                <VisualState x:Name="Collapsed">
+                                    <VisualState.Setters>
+                                        <Setter Target="PART_Indicator.Visibility" Value="Collapsed" />
                                     </VisualState.Setters>
                                 </VisualState>
-                                <!--  TODO  -->
-                                <VisualState x:Name="Disabled" />
                             </VisualStateGroup>
                         </VisualStateManager.VisualStateGroups>
 
-                        <TextBlock HorizontalAlignment="{TemplateBinding HorizontalContentAlignment}"
-                                   VerticalAlignment="{TemplateBinding VerticalContentAlignment}"
-                                   win:AutomationProperties.AccessibilityView="Raw"
-                                   FontFamily="{ThemeResource GripperBarFontFamily}"
-                                   Foreground="{ThemeResource GripperBarForeground}"
-                                   Text="{Binding Orientation, RelativeSource={RelativeSource TemplatedParent}, Converter={StaticResource OrientationToGlyphConverter}}" />
+                        <Rectangle x:Name="PART_Indicator"
```

Comment:
Should we call this (and the property) a `Thumb` instead like a `Scollbar`/`Slider` type part? (More nomenclature from WPF as they didn't really bring the class forward to UWP [think just internal-ish primitive], but it generally means the knob you use to slide around.)

Reply:
Agree! Updated.